source/api/presentation/3.0/index.md

```diff
@@ -1456,32 +1456,26 @@ This section describes the _RECOMMENDED_ request and response interactions for t
 
 Clients _MUST NOT_ construct resource URIs by themselves, instead they _MUST_ follow links from within retrieved descriptions.
 
-Implementation note doesn't belong in spec
-{: .warning}
-In the situation where the JSON documents are maintained in a filesystem with no access to the web server's configuration, then including ".json" on the end of the URI is suggested to ensure that an appropriate `content-type` response header is sent to the client.
-
 
 ###  6.2. Responses
 
-The format for all responses is JSON, and the following sections describe the structure to be returned.
-
-
+The format for all responses is JSON, as described above.
 
-The content-type of the response _SHOULD_ be "application/ld+json" (JSON-LD) with the `profile` parameter given as the context document: `http://iiif.io/api/presentation/3/context.json`.
+The HTTP `Content-Type` header of the response _SHOULD_ have the value "application/ld+json" (JSON-LD) with the `profile` parameter given as the context document: `http://iiif.io/api/presentation/3/context.json`.
 
 ``` none
 Content-Type: application/ld+json;profile="http://iiif.io/api/presentation/3/context.json"
 ```
 {: .urltemplate}
 
-If this cannot be generated due to server configuration details, then the content-type _MUST_ instead be `application/json` (regular JSON).
+If this cannot be generated due to server configuration details, then the content-type _MUST_ instead be `application/json` (regular JSON), without a `profile` parameter.
 
 ``` none
 Content-Type: application/json
 ```
 {: .urltemplate}
 
-The HTTP server _MUST_ follow the CORS requirements, including the `Access-Control-Allow-Origin` and the value of the header _SHOULD_ be `*`.
+The HTTP server _MUST_ follow the CORS requirements, including an `Access-Control-Allow-Origin` header with the value of `*`.
 
```

Comment:
Is a specific origin allowed, and list of specific origins?

Reply:
How about _MUST_ have the header, and value _SHOULD_ be * ?